tests/lambda_functions/vetext_incoming_forwarder_lambda/test_vetext_incoming_forwarder_lambda.py

```diff
@@ -87,7 +95,7 @@ def test_failed_vetext_call_goes_to_sqs(mocker, http_failure_response, event):
 
 @pytest.mark.parametrize('event', [(albInvokedWithoutAddOn), (albInvokeWithAddOn), (sqsInvokedWithAddOn)])
 def test_failed_vetext_call_throws_http_exception(mocker, event):
```

Comment:
How is this function directly testing that HttpException was raised?

Reply:
if exception wasnt raised then retry sqs wouldnt be called.